build-system/tasks/presubmit-checks.js

```diff
@@ -647,19 +647,21 @@ var forbiddenTermsSrcInclusive = {
   '\\.scheduleUnlayout\\(': bannedTermsHelpString,
   'getComputedStyle\\(': {
     message: 'Due to various bugs in Firefox, you must use the computedStyle ' +
-        'helper in style.js.',
+    'helper in style.js.',
     whitelist: [
       'src/style.js',
       'dist.3p/current/integration.js',
     ],
   },
   'decodeURIComponent\\(': {
     message: 'decodeURIComponent throws for malformed URL components. Please ' +
-        'use tryDecodeUriComponent from src/url.js',
+    'use tryDecodeUriComponent from src/url.js',
     whitelist: [
       '3p/integration.js',
       'dist.3p/current/integration.js',
       'examples/pwa/pwa.js',
+      'examples/amp-web-push/publisher-files/helper-frame.js',
```

Comment:
`tryDecodeUriComponent()` is now imported from the AMP framework instead of duplicated.

I added a `gulp build-web-push-publisher-files` build task for generating publisher files that's run with `gulp dist`. So I'm now able to import AMP framework methods in to the two publisher files `helper-frame.js` and `permission-dialog.js` without duplicating the code.